sdks/go/pkg/beam/core/runtime/exec/fn.go

```diff
@@ -23,10 +23,9 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/core/funcx"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/graph"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/typex"
+	"github.com/apache/beam/sdks/go/pkg/beam/core/util/reflectx"
 )
 
-//go:generate specialize --input=callers.tmpl --x=data,universals
-
 // NOTE(herohde) 12/11/2017: the below helpers are ripe for type-specialization,
 // if the reflection overhead here turns out to be significant. It would
```

Comment:
Is this comment still relevant? Seems we're doing all the type specialization now.